Livecheckables/povray.rb

```diff
@@ -1,4 +1,3 @@
 class Povray
-  livecheck :url   => "https://github.com/POV-Ray/povray/releases",
-            :regex => %r{Latest.*?href="/POV-Ray/povray/tree/v([0-9\.]+)}m
+  livecheck :regex => /^v?(\d+(?:\.\d+)+)$/
```

Comment:
Due to the tags like `v3.7.0.20140613`, we'll need to make sure that the last set of digits is a reasonable length (something fewer than 8 digits). Let's just set the upper limit for the last digit(s) to be 4 or fewer, since that seems reasonable.

```suggestion
  livecheck :regex => /^v?(\d+(?:\.\d+)+\.\d{,4})$/
```

Reply:
Nice!